requirements.txt

```diff
@@ -1,7 +1,7 @@
-numpy
-celery
-PyFlakes
+numpy==1.12.1
+celery==4.1.0
+pyflakes==1.6.0
```

Comment:
@RoranDangerous I don't think this package is required it may have already been on your system when you ```pip freeze```ed. please remove.